src/App.css

```diff
@@ -1,38 +1,59 @@
-.App {
-  text-align: center;
+*{
+  margin: 0;
+  padding:0;
+  box-sizing: border-box;
 }
 
-.App-logo {
-  height: 40vmin;
-  pointer-events: none;
-}
-
-@media (prefers-reduced-motion: no-preference) {
-  .App-logo {
-    animation: App-logo-spin infinite 20s linear;
-  }
-}
-
-.App-header {
-  background-color: #282c34;
-  min-height: 100vh;
+.container{
   display: flex;
   flex-direction: column;
-  align-items: center;
-  justify-content: center;
-  font-size: calc(10px + 2vmin);
-  color: white;
+  justify-content: space-between;
+  align-items: center ;
+  height:400px;
+  width:300px;
+  border: black 2px solid;
+  border-radius: 5px;
+  margin:100px;
+  padding:10px;
```

Comment:
don't know if it was intentionally but loved how the css properties are grouped:
flex-properties
height/width
borders
margin/paddings

👏🏻

Reply:
thnx auntie shoniee